src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const smallDiscount = 20;
+  const bigDiscount = 30;
+  const payPerDays = 40;
+  let sum = payPerDays * days;
+
+  if (days >= 3) {
+    sum -= smallDiscount;
```

Comment:
if you hit the spot, you can return a result right here
```suggestion
    return sum - smallDiscount; // like this
```